src/main/groovy/com/blackducksoftware/integration/hub/detect/util/FileFinder.groovy

```diff
@@ -118,36 +118,36 @@ class FileFinder {
         return files
     }
 
-    private File[] findDirectoriesContainingDirectoriesToDepth(final File sourceDirectory, final String directoryPattern, int maxDepth) {
-        findDirectoriesContainingDirectoriesToDepthRecursive(sourceDirectory, directoryPattern, 0, maxDepth);
+    public File[] findDirectoriesContainingDirectoriesToDepth(final File sourceDirectory, final String directoryPattern, int maxDepth) {
```

Comment:
Why did you change this to public?

Reply:
It was called by detect configuration. With the @CompileStatic annotation you cannot access private methods. Switching to @TypeChecked means I can revert this back to a private method